examples/provide-liquidity/utils/pool.ts

```diff
@@ -44,31 +50,38 @@ import { Fee } from '@/hooks';
 
 export const osmosisAssets: Asset[] = [
   ...assets.assets,
-  ...asset_list.assets,
+  ...asset_list.assets.filter((item) => item.display != 'usdc'),
 ].filter(({ type_asset }) => type_asset !== 'ics20');
 
+export const osmosisAssetsList: AssetList[] = [
+  {
+    assets: osmosisAssets,
+    chain_name: 'osmosis',
+  },
+];
+// console.log(asset_list, assets, osmosisAssetsList);
 export const getOsmoAssetByDenom = (denom: CoinDenom): Asset => {
```

Comment:
I think ideally, we don't rebuild new methods here, we should either

1. use the chain-registry/utils and pass the word osmosis for chain_name, or use the chain-registry/client ChainRegistryClient class

AND/OR

2. use https://github.com/osmosis-labs/osmojs/tree/main/packages/math which I think has a lot of these methods, also

Reply:
e, I tested it.

```
import { asset_list, assets } from '@chain-registry/osmosis';

export const osmosisAssets: Asset[] = [
  ...assets.assets,
  ...asset_list.assets.filter((item) => item.display != 'usdc'),
].filter(({ type_asset }) => type_asset !== 'ics20');

export const osmosisAssetsList: AssetList[] = [
  {
    assets: osmosisAssets,
    chain_name: 'osmosis',
  },
];
```

The previous developers had a reason to merge assets and asset_list. If you only use asset_list, some assets might not be found.

if not merge
```

import { asset_list, assets } from '@chain-registry/osmosis';
export const osmosisAssets: Asset[] =  assets.filter(({ type_asset }) => type_asset !== 'ics20');
const osmosisAssetsList = [asset_list];

```
<img width="711" alt="image" src="https://github.com/cosmology-tech/create-cosmos-app/assets/14268015/46342983-e7cb-49f6-b1b1-22030acfecd1">

Reply:
fix: https://github.com/stark-tech-space/create-cosmos-app/pull/3/commits